src/main/java/gg/solarmc/kitpvp/kill/damage/DamageMap.java

```diff
@@ -1,13 +1,13 @@
-package gg.solar.kitpvp;
+package gg.solarmc.kitpvp.kill.damage;
 
-import me.aurium.beetle.core.config.DataHolder;
+import gg.solarmc.kitpvp.KitpvpPlugin;
 import org.bukkit.entity.Player;
 
 import java.util.HashMap;
 import java.util.Map;
 
 /**
- * Map of all players to their respective holders.
+ * Map of all players to their respective holders. No need for concurrency.
```

Comment:
I think it's pretty clear that this plugin is not made to operate concurrently.

I take an all-or-nothing approach to concurrency where entire projects are either completely thread safe or not at all. In such cases you don't have to repeat the same documentation.

Reply:
i wrote that there because my brain was debating whether i needed that class to be concurrent.

Reply:
Besides, there isn't really a point into making a plugin fully threadsafe